assets/js/just-the-docs.js

```diff
@@ -454,7 +454,19 @@ jtd.getTheme = function() {
 
 jtd.setTheme = function(theme) {
   var cssFile = document.querySelector('[rel="stylesheet"]');
-  cssFile.setAttribute('href', '{{ "assets/css/just-the-docs-" | absolute_url }}' + theme + '.css');
+  cssFile.setAttribute('href', '{{ "assets/css/just-the-docs-" | relative_url }}' + theme + '.css');
+}
+
+// Scroll site-nav to ensure the link to the current page is visible
+
+function scrollNav() {
```

Comment:
Perhaps by changing the function name to something in the lines of `ensureScrollNavLinksAreVisible()` would make it more explicit, removing the need for a comment here

(Naming Things Is Hard™) 😄